cmd/addvalidators.go

```diff
@@ -93,11 +92,23 @@ func bindAddValidatorsFlags(cmd *cobra.Command, config *addValidatorsConfig) {
 }
 
 func runAddValidatorsSolo(ctx context.Context, conf addValidatorsConfig) (err error) {
-	// Read lock file to load cluster manifest
-	cluster, _, err := loadClusterManifest(conf)
-	if err != nil {
-		return err
+	var cluster *manifestpb.Cluster
+
+	if conf.TestConfig.Manifest != nil {
+		cluster = conf.TestConfig.Manifest
+	} else if conf.TestConfig.Lock != nil {
+		cluster, err = manifest.NewClusterFromLock(*conf.TestConfig.Lock)
+		if err != nil {
+			return err
+		}
+	} else {
+		// Read lock file to load cluster manifest
+		cluster, _, err = loadClusterManifest(conf.ManifestFile, conf.Lockfile)
+		if err != nil {
+			return err
+		}
 	}
```

Comment:
nit: we can probably move this logic to the `loadClusterManifest()` function. The function then should return a cluster manifest either from TestConfig or by loading from disk.

Reply:
Yeah maybe we can move TestConfig in a separate file (it's in `addvalidators.go` now) and have `cmd` share it.